shell/components/LabelsPSA.vue

```diff
@@ -0,0 +1,72 @@
+<script>
```

Comment:
Optional - I haven't seen this running yet, but low bar solution would be to just pass the `mode` through to the component's of the PodSecurityAdmission component. If the mode is something not editable the controls will then be disabled.

Reply:
Only some keys are disabled, not all.
All the keys are put together as all the labels are together as well.
Please refer to the design.

Reply:
I've seen the design now, most of the conversation is more relevant there (given that we can hopefully use the existing `Labels` component

Reply:
This has been also discussed and we cannot use the existing `Labels` component, as this configuration is exceptional to Namespace only. Your option was already evaluated as first approach, but the component has nothing relevant to be reused and is rather complicating the view. I opted for keeping the tabs separated to avoid issues.